Components/GH_terrainArrays.cs

```diff
@@ -58,8 +60,10 @@ protected override void SolveInstance(IGH_DataAccess DA)
             var analysisPoints = GetPointsAnalysis(rectTerrain);
 
             var rayTerrain = HitPointsHeight(gridPoints, terrainToMesh);
-            Rhino.Geometry.Mesh ProbMesh = CreateMeshFromGridPoints(HitPoints(analysisPoints, terrainToMesh),201,201);
-      
+            //Rhino.Geometry.Mesh ProbMesh = CreateMeshFromGridPoints(HitPoints(analysisPoints, terrainToMesh),201,201);
+            Rhino.Geometry.Mesh ProbMesh = CreateMeshFromGridPoints(HitPoints(analysisPoints, terrainToMesh), 200, 200);
```

Comment:
@MikaelSpinnangr I think we need 201x201 to perform the operation. Was there any reason that you changed this?